tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "jsx": "preserve",
     "sourceMap": true,
     "strict": true,
+    "skipLibCheck": true,
```

Comment:
Definitely less concerned about this here than on https://github.com/zendeskgarden/react-components/pull/1175.